apps/core/src/migration/migration.e2e.spec.ts

```diff
@@ -195,7 +194,47 @@ describe('MigrationModule', () => {
       expect(migrateSpy).toHaveBeenCalled();
       expect(createSpy).toHaveBeenCalledWith({
         name: testDashboard.dashboardName,
-        description: testDashboard.dashboardDescription,
+        description: applicationDashboardDescription,
+        definition: expectedDefinition,
+        sitewiseMonitorId: testDashboard.dashboardId,
+      });
+
+      expect(response.statusCode).toBe(202);
+
+      const status = await waitForStatus();
+      expect(status).toEqual({ status: Status.COMPLETE });
+    });
+
+    test('handles dashboardName when Monitor dashboard name is at the max length', async () => {
+      sitewiseMock.on(ListPortalsCommand).resolves(testPortals);
+      sitewiseMock.on(ListProjectsCommand).resolves(testProjects);
+      sitewiseMock.on(ListDashboardsCommand).resolves(testDashboards);
+
+      const longDashboardName =
+        'testtesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttest';
```

Comment:
if we like the string to fill the name, we can repeat it programmatically by `'test'.repeat(DASHBOARD_NAME_MAX_LENGTH/4).padEnd(DASHBOARD_NAME_MAX_LENGTH, '-')` for better maintenance.

Reply:
ah good call, I see Tracy does something similar for the maxLength tests, I'll update

Reply:
updated